EventFilter/EcalRawToDigi/python/ecalDigis_cff.py

```diff
@@ -41,3 +45,38 @@
     # run the ECAL unpacker on CPU, or copy the ECAL digis from GPU to CPU and covert them from SoA to legacy format
     ecalDigis
 ))
+
+# process modifier to run alpaka implementation
+from Configuration.ProcessModifiers.alpaka_cff import alpaka
+
+# ECAL conditions used by the portable unpacker
+from EventFilter.EcalRawToDigi.ecalElectronicsMappingHostESProducer_cfi import ecalElectronicsMappingHostESProducer
+
+# alpaka ECAL unpacker
+from EventFilter.EcalRawToDigi.ecalRawToDigiPortable_cfi import ecalRawToDigiPortable as _ecalRawToDigiPortable
+ecalDigisPortable = _ecalRawToDigiPortable.clone()
+
+from EventFilter.EcalRawToDigi.ecalDigisFromPortableProducer_cfi import ecalDigisFromPortableProducer as _ecalDigisFromPortableProducer
+
+# replace the SwitchProducer branches with a module to copy the ECAL digis from the accelerator to CPU and covert them from SoA to legacy format
+alpaka.toModify(ecalDigis,
+    cpu = _ecalDigisFromPortableProducer.clone(
+        digisInLabelEB = 'ecalDigisPortable:ebDigis',
+        digisInLabelEE = 'ecalDigisPortable:eeDigis',
+        produceDummyIntegrityCollections = True
+    ),
+    cuda = _ecalDigisFromPortableProducer.clone(
+        digisInLabelEB = 'ecalDigisPortable:ebDigis',
+        digisInLabelEE = 'ecalDigisPortable:eeDigis',
+        produceDummyIntegrityCollections = True
+    )
+)
+
+alpaka.toReplaceWith(ecalDigisTask, cms.Task(
+    # ECAL conditions used by the portable unpacker
+    ecalElectronicsMappingHostESProducer,
+    # run the portable ECAL unpacker
+    ecalDigisPortable,
+    # run the ECAL unpacker on CPU, or copy the ECAL digis from GPU to CPU and covert them from SoA to legacy format
```

Comment:
```suggestion
    # run the ECAL unpacker on CPU, or copy the ECAL digis from GPU to CPU and convert them from SoA to legacy format
```

Reply:
Actually, in the Alpaka case I think that the comment is wrong ?

Reply:
Yes indeed.